sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/crypto/_models.py

```diff
@@ -18,11 +18,11 @@ class DecryptResult:
     :param bytes plaintext: The decrypted bytes
     """
 
-    def __init__(self, key_id, algorithm, plaintext):
-        # type: (str, EncryptionAlgorithm, bytes) -> None
-        self.key_id = key_id
+    def __init__(self, algorithm, plaintext, key_id=None):
+        # type: (EncryptionAlgorithm, bytes, Optional[str]) -> None
```

Comment:
For each of these types we should avoid changing the signature and just update type hints (as much as we both dislike accepting `None` for a required parameter):
```suggestion
    def __init__(self, key_id, algorithm, plaintext):
        # type: (Optional[str], EncryptionAlgorithm, bytes) -> None
```